playwright.config.ts

```diff
@@ -23,39 +25,39 @@ module.exports = defineConfig({
       name: "chromium",
       use: { ...devices["Desktop Chrome"],
         channel: "chrome",
-        headless: false,
+        headless: headless,
```

Comment:
are you sure that you mean headless: headless and not headless: true
headless: "headless" is possible I suppose, but I've very skeptical about what you've got here

Reply:
I have changed this variable to headlessMode. The default value is true.